src/applications/personalization/profile/tests/e2e/notification-settings/feature-toggles.cypress.spec.js

```diff
@@ -1,5 +1,5 @@
 import mockCommunicationPreferences from '@@profile/tests/fixtures/communication-preferences/get-200-maximal.json';
-import { generateFeatureToggles } from 'applications/personalization/profile/mocks/endpoints/feature-toggles';
+import { generateFeatureToggles } from '~/applications/personalization/profile/mocks/endpoints/feature-toggles';
```

Comment:
if you replace this with `'@@profile/mocks/endpoints/feature-toggles'` you can reduce this cypress test by one second lol